modules/UI.js

```diff
@@ -0,0 +1,49 @@
+import Score from './addScore.js';
```

Comment:
- [ ] For a better project structure, I recommend moving the `modules` folder to `/src` so your project is well organized.